llvm/test/Transforms/InstCombine/fsh.ll

```diff
@@ -722,6 +722,134 @@ define i32 @fsh_orconst_rotate(i32 %a) {
   ret i32 %t2
 }
 
+define i32 @fsh_rotate_25(i8 %x, i32 %y) {
```

Comment:
It would be nice to explicitly mark with a comment the tests where the simplification didn't not trigger anyway.

Reply:
+1 to indicate negative tests

Als, you can add the tests as a stacked first commit so the second commit in the patch can show the test changes